src/neo-vm/ExecutionEngine.cs

```diff
@@ -1242,6 +1247,10 @@ internal protected void ExecuteNext()
                 {
                     State = VMState.FAULT;
                 }
+                finally
+                {
+                    Trace();
```

Comment:
You don't make any difference between different methods?

Reply:
I don't understand what you're asking